tests/integration/protostar_fixture.py

```diff
@@ -208,7 +205,9 @@ def format_with_output(
 
         return summary, output
 
-    def create_files(self, relative_path_str_to_file: Dict[str, Union[str, Path]]) -> None:
+    def create_files(
+        self, relative_path_str_to_file: Dict[str, Union[str, Path]]
+    ) -> None:
```

Comment:
nothing changes in this file except formatting?

Reply:
black formats that file for me actually, dk really why

Reply:
maybe outdated black version you have